.github/workflows/copy_to_gitbook_repo.yml

```diff
@@ -0,0 +1,45 @@
+name: Copy file to repository synced with Gitbook
+
+on:
+  workflow_call:
+    inputs:
+      source_file:
+        required: true
+        type: string
+      destination_folder:
+        type: string
+      rename:
+        type: string
+      action_name:
+        type: string
+    secrets:
+      access_token:
+        required: true
+
+jobs:
+  copy_file:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
there should already be actions/checkout@v4 , is there any reason to stick with the older version?

https://github.com/actions/checkout